packages/mendel-config/src/bundle-config.js

```diff
@@ -23,6 +36,20 @@ function BundleConfig(id, options, config) {
         options,
         ['generator', 'outfile', 'outlet', 'entries', 'require', 'external']
     );
+
+    Object.keys(this.options).forEach((optionKey) => {
+        if (supportedOptions.indexOf(optionKey) === -1) {
+            unsupportedOptions.push(optionKey);
+        }
+    });
+
+    if (unsupportedOptions.length > 0) {
+        console.log([
+            '[Config][WARN] Following options are not supported for bundle: ' + id + '.\n',
+            unsupportedOptions.join(', '),
+        ].join(' '));
+    }
+
     this.options = undash(this.options);
 
     BundleConfig.validate(this);
```

Comment:
There is a validator and this is the line that invokes the validation. Can whitelist of keys be enforced in the validator instead of having sprinkle of config here and there? (this is a recommendation and if you think it sucks, just ignore this suggestion)